orquesta/utils/context.py

```diff
@@ -40,11 +40,19 @@ def set_current_task(context, task):
     return ctx
 
 
-def set_current_item(context, item):
+def copy_context(context):
     if context and not isinstance(context, dict):
         raise TypeError("The context is not type of dict.")
 
     ctx = json_util.deepcopy(context) if context else dict()
+    return ctx
+
+
+def set_current_item(context, item):
+    if context and not isinstance(context, dict):
+        raise TypeError("The context is not type of dict.")
+
+    ctx = {**context}
```

Comment:
This seems uneccessary, just use context that is incoming correct?  Wouldn't this do  a shallow copy?